spec/models/book_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Actually, I like single quote as well, but one of my team currently just stick with double quotes all the time. Either way should be fine with me.